src/bp-core/bp-core-theme-compatibility.php

```diff
@@ -1055,3 +1055,74 @@ function bp_theme_compat_is_block_theme() {
 
 	return isset( $theme->is_block_theme ) && $theme->is_block_theme;
 }
+
+/**
+ * Registers the `buddypress` theme feature.
+ *
+ * @since 14.0.0
+ */
+function bp_register_buddypress_theme_feature() {
+	register_theme_feature(
+		'buddypress',
+		array(
+			'type'        => 'array',
+			'variadic'    => true,
+			'description' => __( 'Whether the Theme supports BuddyPress and possibly BP Modern features', 'buddypress' ),
+		)
+	);
+}
+add_action( 'bp_init', 'bp_register_buddypress_theme_feature' );
+
+/**
+ * Filters the WP theme support API so that it can be used to check whether the
+ * current theme has global BuddyPress and/or BP Component specific support.
+ *
+ * Please do not use in your plugins or themes.
+ *
+ * @since 14.0.0
+ * @access private
+ *
+ * @param bool   $supports Whether the active theme supports the given feature. Default false.
+ * @param array  $args     Array of arguments for the feature.
+ * @param string $feature  The theme feature.
```

Comment:
it looks like this can be an array too? Maybe set `array|string` or `mixed`?
```suggestion
 * @param mixed $feature  The theme feature.
```

Reply:
This function is used to filter WordPress `current_theme_supports-{$feature}` hook. `$feature` should always be a string: https://developer.wordpress.org/reference/hooks/current_theme_supports-feature/

Reply:
I spoke too quick, you're right!